src/r2mm/launching/runners/linux/SteamGameRunner_Linux.ts

```diff
@@ -68,8 +69,10 @@ export default class SteamGameRunner_Linux extends GameRunnerProvider {
 
         LoggerProvider.instance.Log(LogSeverity.INFO, `Steam directory is: ${steamDir}`);
 
+        const steamCmd = steamDir.indexOf(path.join(homedir(), '.var', 'app', 'com.valvesoftware.Steam')) == 0 ? `flatpak run --branch=stable --arch=x86_64 --command=/app/bin/steam-wrapper --file-forwarding --filesystem=home/.config/r2modmanPlus-local com.valvesoftware.Steam` : `"${steamDir}/steam.sh"`;
```

Comment:
A few questions which could be good to cover as a comment block above the line:

1. Why is `--branch=stable` needed?
2. What is this doing? `--command=/app/bin/steam-wrapper`
3. What does `--filesystem=home/.config/r2modmanPlus-local com.valvesoftware.Steam` do?
4. File forwarding?

Is there a way we can determine if it's installed using flatpak beforehand rather than essentially doing: `if exe else flatpak`?

Reply:
Yeah, these arguments were necessary when running r2modman **as a** flatpak, and i forgot to reevaluate them for just steam flatpak compatibility. I will update my PR, only `flatpak run com.valvesoftware.Steam` is necessary.

As for determinating beforehand if steam is flatpak, I'll have to have a better look at the code. Do you have any suggestions of where I could make this check?

Reply:
Oh, I found this LinuxGameDirectoryResolver, there is already checks like isProtonGame in that class. I think I'll put a "isFlatpakSteam" there